client/my-sites/checkout/composite-checkout/types/wpcom-store-state.ts

```diff
@@ -531,7 +531,6 @@ export function prepareDomainContactValidationRequest(
 			last_name: details.lastName?.value,
 			organization: details.organization?.value,
 			email: details.email?.value,
-			alternate_email: details.alternateEmail?.value,
```

Comment:
If we're removing `alternate_email` from this object and we expect it to never be set, we might want to remove it from the `DomainContactValidationRequest` type (and maybe the `ContactValidationRequestContactInformation`/`ContactValidationResponseMessages` types too?).

Should we also remove it from `formatDomainContactValidationResponse()`?

Reply:
Thanks for the review 🙏

You're right, the intent was to remove this field altogether, since it doesn't serve any other purpose than for the Google Workspace checkout form. I've gone ahead and removed the remaining references!